frontend/fingertips-frontend/components/charts/LineChartOverTime/helpers/lineChartOverTimeData.ts

```diff
@@ -58,13 +62,29 @@ export const lineChartOverTimeData = (
     healthData.areaHealthData?.at(0)?.healthData?.at(0)?.datePeriod?.type ??
     PeriodType.Calendar;
   const frequency = healthData.frequency ?? Frequency.Annually;
+  const reportingPeriod =
+    selectedReportingPeriod ??
+    reportingPeriodOptions[0] ??
+    ReportingPeriod.Yearly;
+
+  const reportingPeriodFlag =
+    reportingPeriod.toLowerCase() === frequency.toLowerCase() ||
+    (reportingPeriod.toLowerCase() ===
+      reportingPeriodLabelOrder[ReportingPeriod.Yearly]?.label.toLowerCase() &&
+      frequency === Frequency.Annually) ||
+    (reportingPeriod.toLowerCase() ===
+      reportingPeriodLabelOrder[
+        ReportingPeriod.CumulativeQuarterly
+      ]?.label.toLowerCase() &&
+      frequency === Frequency.Quarterly);
```

Comment:
Will be refactored in follow on PR. This logic will need to be replicated per chart